src/dev/env.md

```diff
@@ -7,8 +7,22 @@ Variables prefixed with `YKD_` are intended for debugging only. Most (if not
 all) of the debugging variables introduce extra computation that slows down
 program execution.
 
+Some variables are conditionally compiled in when the containing crate is built
```

Comment:
Should we make clearer that this is an *internal* feature?

[Also s/compiled in/compiled/]

Reply:
Added prose on the internalness of it.

I meant `compiled-in`. Try [a208b12](https://github.com/ykjit/ykdocs/pull/35/commits/a208b12ddc88a74f797b750789fcda646f5806b9).